sysconfig/ubuntu-18.04/packages.yaml

```diff
@@ -0,0 +1,107 @@
+packages:
+    autoconf:
+        paths:
+            autoconf@2.69: /usr
+        version: [2.69]
+    automake:
+        paths:
+            automake@1.15.1: /usr
+        version: [1.15.1]
+    bison:
+        paths:
+            bison@3.0.4: /usr
+        version: [3.0.4]
+    boost:
+        paths:
+            boost@1.65.1: /usr
+        version: [1.65.1]
+    bzip2:
+        paths:
+            bzip2@1.0.6: /usr
+        version: [1.0.6]
+    cmake:
+        paths:
+            cmake@3.10.2: /usr
+        version: [3.10.2]
+    diffutils:
+        paths:
+            diffutils@3.6: /usr
+        version: [3.6]
+    findutils:
+        paths:
+            findutils@4.6.0: /usr
+        version: [4.6.0]
+    flex:
+        paths:
+            flex@2.6.2: /usr
+        version: [2.6.2]
+    hdf5:
+        paths:
+            hdf5@1.10: /usr/lib/x86_64-linux-gnu/hdf5/serial
+        version: [1.10]
```

Comment:
I will explicitly add ~mpi for hdf5. Note that our packages are depend on parallel hdf5 now. So could you check parallel hdf5 with corresponding mpi ?